pulp_puppet_handlers/pulp_puppet/handlers/puppet.py

```diff
@@ -428,7 +419,7 @@ def bind(conduit, binding, options):
         :rtype:  BindReport
         """
         repo_id = binding['repo_id']
-        logger.info('binding to repo %s' % repo_id)
+        logger.info(_('binding to repo %s') % repo_id)
```

Comment:
FWIW, logger.info(_('binding to repo %s'), repo_id) is considered to better because the variable substitution only happens if the log level meets the criteria for actually logging the message.